src/Components/CustomAlert.js

```diff
@@ -1,25 +1,60 @@
 import React from "react"
-import { Stack, Alert } from "@mui/material"
+import { Alert } from "@mui/material"
+import { Slide } from "@mui/material"
 
 let StyledAlert = (props) => {
-	console.log("styledAlert props", props)
-	return (
-		<Stack spacing={2}>
-			<Alert
-				variant="filled"
-				severity="error"
-				sx={{ display: `${props.errorVisible}` }}
-			>
-				{props.message}
-			</Alert>
-		</Stack>
-	)
+	let variableForSlideInProp = props.errorVisible !== "none"
+	let errorDirection = "up"
+	if (props.errorVisible === "userCreated") {
+		//returns succuss alert of user being created
+		console.log("alert props", props)
+		return (
+			<>
+				<Slide
+					direction={`${errorDirection}`}
+					in={variableForSlideInProp}
+					mountOnEnter
+					unmountOnExit
+				>
+					<Alert
+						variant="filled"
+						severity="success"
+						sx={{ display: `${props.errorVisible}` }}
+					>
+						{`User successfully created! Please log in with email '${props.userDataObj.email}' and password '${props.userDataObj.password}'`}
+					</Alert>
+				</Slide>
+			</>
+		)
+	} else {
+		//returns error alert of user not being created
+		return (
+			<>
+				<Slide
+					direction={`${errorDirection}`}
+					in={variableForSlideInProp}
+					mountOnEnter
+					unmountOnExit
+				>
+					<Alert
+						variant="filled"
+						severity="error"
+						sx={{ display: `${props.errorVisible}` }}
+					>
+						{props.message}
+					</Alert>
+				</Slide>
+			</>
+		)
+	}
 }
-export default function ErrorAlert(props) {
+export function ErrorAlert(props) {
 	return (
 		<StyledAlert
 			message={props.message}
 			errorVisible={props.errorVisible}
+			setErrorVisible={props.setErrorVisible}
+			userDataObj={props.userDataObj}
 		></StyledAlert>
```

Comment:
Seems like you are just passing things through here, so you could just use StyledAlert directly.